api-catalog-ui/frontend/src/components/Swagger/SwaggerUI.jsx

```diff
@@ -150,7 +175,6 @@ export default class SwaggerUI extends Component {
                         presets: [SwaggerUi.presets.apis],
                         requestSnippetsEnabled: true,
                         plugins: [this.customPlugins, AdvancedFilterPlugin, CustomizedSnippedGenerator(codeSnippets)],
-                        filter: true,
```

Comment:
Is it ok having it deleted here?

Reply:
it looks a problem of GH. Actually this line is not removed